cmd/protoc-gen-docs/htmlGenerator.go

```diff
@@ -762,7 +762,7 @@ func (g *htmlGenerator) generateComment(loc protomodel.LocationDescriptor, name
 				typeName := match[end+2 : len(match)-1]
 
 				if o, ok := g.model.AllDescByName["."+typeName]; ok {
-					return g.linkify(o, linkName)
+					return g.linkify(o, linkName, false)
```

Comment:
no idea what the false signifies.

Reply:
Whether to render a type as fully dotted (google.protobuf.Duration) or trim it down (Duration)